cip25-wasm/src/lib.rs

```diff
@@ -710,36 +708,40 @@ impl From<MetadataDetails> for core::MetadataDetails {
     }
 }
 
+/// A String of at most 64 bytes.
+/// This is to conform with Cardano metadata restrictions.
 #[wasm_bindgen]
-
 #[derive(Clone, Debug)]
 pub struct String64(core::String64);
 
 #[wasm_bindgen]
-
 impl String64 {
-    pub fn to_bytes(&self) -> Vec<u8> {
-        ToBytes::to_bytes(&self.0)
+    pub fn new(s: &str) -> Result<String64, JsError> {
+        core::String64::new(s).map(Self).map_err(Into::into)
     }
 
-    pub fn from_bytes(data: Vec<u8>) -> Result<String64, JsValue> {
-        FromBytes::from_bytes(data).map(Self).map_err(|e| JsValue::from_str(&format!("from_bytes: {}", e)))
+    pub fn to_str(&self) -> String {
+        self.0.to_str().to_owned()
+    }
+
+    pub fn to_bytes(&self) -> Vec<u8> {
```

Comment:
Do we even want to be providing these to/from bytes for all parts of this spec? I don't know why you would want to do it for anything but the root `CIP25Metadata` type.